cli-requirejs-demo/aurelia_project/aurelia.json

```diff
@@ -1,98 +1,92 @@
     "targets": [
       {
-        "id": "web",
-        "displayName": "Web",
+        "port": 9000,
         "index": "index.html",
         "baseDir": ".",
         "output": "scripts"
       }
     ],
     "options": {
       "minify": "stage & prod",
-      "sourcemaps": "dev & stage"
+      "sourcemaps": "dev & stage",
+      "rev": "prod",
+      "cache": "dev & stage"
     },
     "bundles": [
       {
         "name": "app-bundle.js",
         "source": [
-          "[**/*.js]",
-          "**/*.{css,html}"
+          "**/*.{js,json,css,html}"
         ]
       },
       {
         "name": "vendor-bundle.js",
         "prepend": [
-          "node_modules/bluebird/js/browser/bluebird.core.js",
-          "node_modules/aurelia-cli/lib/resources/scripts/configure-bluebird.js",
+          "node_modules/regenerator-runtime/runtime.js",
+          "node_modules/promise-polyfill/dist/polyfill.min.js",
           "node_modules/requirejs/require.js"
         ],
         "dependencies": [
-          "aurelia-binding",
           "aurelia-bootstrapper",
-          "aurelia-dependency-injection",
-          "aurelia-event-aggregator",
-          "aurelia-framework",
-          "aurelia-history",
-          "aurelia-history-browser",
-          "aurelia-http-client",
-          "aurelia-loader",
           "aurelia-loader-default",
-          "aurelia-logging",
-          "aurelia-logging-console",
-          "aurelia-metadata",
-          "aurelia-pal",
           "aurelia-pal-browser",
-          "aurelia-path",
-          "aurelia-polyfills",
-          "aurelia-route-recognizer",
-          "aurelia-router",
-          "aurelia-task-queue",
-          "aurelia-templating",
-          "aurelia-templating-binding",
-          "text",
-          "jquery",
-          "moment",
-          "moment-mini",
-          "text-encoding-utf-8",
-          {
-            "name": "i18next",
-            "path": "../node_modules/i18next/dist/umd",
-            "main": "i18next"
-          },
-          {
-            "name": "aurelia-i18n",
-            "path": "../node_modules/aurelia-i18n/dist/amd",
-            "main": "aurelia-i18n"
-          },
-          {
-            "name": "i18next-xhr-backend",
-            "path": "../node_modules/i18next-xhr-backend/dist/umd",
-            "main": "i18nextXHRBackend"
-          },
-          {
-            "name": "aurelia-templating-resources",
-            "path": "../node_modules/aurelia-templating-resources/dist/amd",
-            "main": "aurelia-templating-resources"
-          },
-          {
-            "name": "aurelia-templating-router",
-            "path": "../node_modules/aurelia-templating-router/dist/amd",
-            "main": "aurelia-templating-router"
-          },
           {
             "name": "aurelia-testing",
-            "path": "../node_modules/aurelia-testing/dist/amd",
-            "main": "aurelia-testing",
             "env": "dev"
           },
+          "text",
           {
```

Comment:
we still need bootstrap configured here manually, as the default main is not including tooltip.js which by itself is required by popover.js. So this workaround makes sure we point to the right file.